testdata/networking/centos_latest_admin.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: centos-pod
+  labels:
+    name: centos-pod
+spec:
+  containers:
+  - name: centos-pod
+    command: ["/bin/bash", "-c"]
+    args:
+        ["yum install -y iptables && sleep inf"]
+    image: centos:latest
```

Comment:
image need to push to qe registry

Reply:
Its fine if we keep this with ```Disonnected:No```. Okay to keep regression on connected clusters. WDYT

Reply:
team require our test image need to from QE registry, maybe consider in case the image is gone for public in future. :)

Reply:
Yea we should do that. Great suggestion @zhaozhanqi.